data/sheragi/the emerald sword.txt

```diff
@@ -0,0 +1,854 @@
+			`	"Unless you have a jump drive," Rakei adds. "And you do have one, don't you?"`
+			choice
+				`	"I used to, but not anymore"`
+					goto notanymore
+				`	"I do."`
+			`	"This is perfect! Do you think you can get another one? If we're going there, we're going to need one to install on the Emerald Sword and bring it back home."`
+				goto return
+			label notanymore
+			`	"You sold it?" she replies incredulously. "How can you sell such a priceless piece of technology? Well, I hope there is a way to get more of them without having one, because we're going to need two for the next stage. One for your ship, and another one for the Emerald Sword."`
+			label return
+			`	"What do I get in return?" you ask.`
+			`	"The ship, of course," Albert responds. "You've helped me so much, and you went to hell and back to get this black box decoded. We're going to spend a day or so studying the ship after we get it, making a blueprint and all that, but afterwards, it's all yours to keep! Anyway, I just can't wait to see what that ship even looks like. Give us some time to assemble a crew to help refurbish the ship, and in the meantime, get us a jump drive. Come back to us once you have it."`
+				accept
+	
+	to complete
+		never
+
+mission "Sheragi Archaeology: The Emerald Sword 3"
+	landing
+	name "The Emerald Sword"
+	description "Capture the Emerald Sword."
+	source "Zug"
+	destination "Echoes of Tragedy"
+	cargo "Outfitting Equipment" 50
+	passengers 20
+	blocked `You arrive to make the preparations to capture the Emerald Sword, but Albert stops you. "Captain, we're going to need 50 tons of cargo to take our materials and the jump drive, and 20 additional crew to help, including us. I'd also advise that you save enough bunks to hire some additional crew. The Emerald Sword is an alien ship, so we have no idea how many people we will need to man it. I would recommend erring on the side of caution by bringing along 100 or so extra crew members."`
+	
+	to offer
+		has "Sheragi Archaeology: The Emerald Sword 2: active"
+	
+	on offer
+		require "Jump Drive" 2
+		conversation
+			`You have a jump drive in your possession. Would you like to give it to Albert's crew?`
+			choice
+				`    (Yes.)`
+				`    (No.)`
+					defer
+			`	You give Albert a call, and arrange to meet with him to plan the next step. He shows up at your ship, greets you, and starts loading in some equipment along with the jump drive you brought. He has assembled a crew of around 20 people, including him and the three Hai archaeologists, now comfortable to let go of their disguise once they're back in your ship.`
```

Comment:
`now comfortable to let go of their disguise once`
This wording's a little strange. Maybe `now without a disguise since`?